### PR TITLE
feat: add keyboard shortcut to quit the app on Windows

### PR DESCRIPTION
**Pull request by ykris45**
based on @IGassmann comment to my PR earlier ,

### CODE REVIEW

**Comment by ykris45** [comment]
image
image
when i press ctrl+q app closed

**Comment by IGassmann** [comment]
@ykris45, Please format your code with Prettier: https://github.com/lbryio/lbry-app/blob/master/CONTRIBUTING.md#code-formatting

Also, I would recommend you to use a text editor and not a word processor like WordPad to develop. A simple to use and a good one is: https://code.visualstudio.com/

**Comment by ykris45** [comment]
@IGassmann that is correct bro 
image

**Comment by ykris45** [comment]
or i use VS Code Prettier Formatter guys

**Comment by ykris45** [comment]
i fixed link to this pr on changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
   * LBRY URLs not working on Linux ([#1120](https://github.com/lbryio/lbry-app/issues/1120))
   * Fix Windows notifications not showing ([#1145](https://github.com/lbryio/lbry-app/pull/1145))
   * Fix export issues ([#1163](https://github.com/lbryio/lbry-app/pull/1163))
-
+  * Fix Keyboard Shortcut to Quit APP ([#1163](https://github.com/lbryio/lbry-app/pull/1202))
```
Comment (IGassmann) [inline CHANGELOG.md]: Please rename this as "Add keyboard shortcut to quit the app on Windows" under an unreleased section and not under "Fix".

File: src/main/menu/setupBarMenu.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ export default () => {
             }
           },
         },
+        {
+          label: 'Close',
+          accelerator: 'CmdOrCtrl+Q',
+          role: 'quit',
+        },
```
Comment (IGassmann) [inline src/main/menu/setupBarMenu.js]: This shouldn't be under the help menu. Add it to a new submenu "File".

File: src/main/menu/setupBarMenu.js [hunk 3]
```diff
@@ -42,6 +42,11 @@ export default () => {
             }
           },
         },
+        {
+          label: 'Close',
```
Comment (IGassmann) [inline src/main/menu/setupBarMenu.js]: Remove the label "Close" as it should be automatically specified by the role.

File: CHANGELOG.md (hunk 1 above)
Comment (ykris45) [reply CHANGELOG.md]: @IGassmann i cannot see where a unreleased section

Comment (IGassmann) [reply CHANGELOG.md]: You have to add one.

Comment (ykris45) [reply CHANGELOG.md]: https://github.com/lbryio/lbry-app/pull/1202/commits/572d68d880d2d216d74df66662600e59d439a54e done

Comment (IGassmann) [reply CHANGELOG.md]: It also needs to be under an Added subsection. Also, notice that the pull request number doesn't match the link number.

**Comment by ykris45** [comment]
i think this mission completed

File: src/main/menu/setupBarMenu.js [hunk 4]
```diff
@@ -27,7 +27,11 @@ export default () => {
     },
     {
       role: 'window',
-      submenu: [{ role: 'minimize' }, { role: 'close' }],
+      submenu: [
+        { role: 'minimize' },
+        { role: 'close' },
+        { role: 'quit', accelerator: 'CmdOrCtrl+Q' },
```
Comment (IGassmann) [inline src/main/menu/setupBarMenu.js]: This has to be under a submenu called ”File" and not the window one.

Comment (ykris45) [reply src/main/menu/setupBarMenu.js]: @IGassmann like this guys
{
role: 'file',
      submenu: [{ role: 'quit', accelerator: 'CmdOrCtrl+Q' }],
    },

Comment (ykris45) [reply src/main/menu/setupBarMenu.js]: or do you mean submenu label : 'file'

**Comment by ykris45** [comment]
image
LGTM @tzarebczan ...

File: CHANGELOG.md [hunk 5]
```diff
@@ -44,7 +49,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
...
```
Comment (IGassmann) [inline CHANGELOG.md]: Please remove this line.

**Pull request merged**